Item/Item.module.less

```diff
@@ -105,4 +107,12 @@
 			}
 		});
 	});
+
+	&:global(.spottable):active,
+	:global(.spottable):active & {
```

Comment:
Agate now supports a `.active()` mixin, that works just like `.focus()` above.

I also suspect that the `.active()` mixin could be inserted just below the `.focus()` mixin and not include its own `.applySkins`. This will keep all of the skin stuff together and the state stuff inside that.